packages/core/src/utils/collect-updates/collect-updates.ts

```diff
@@ -44,16 +45,14 @@ export function collectUpdates(
       : new Map(filteredPackages.map(({ name }) => [name, packageGraph.get(name)]));
 
   let committish = commandOptions.since;
-  const tagPattern = isIndependent ? '*@*' : '';
+  const tagPattern = describeTag ? describeTag : isIndependent ? '*@*' : '*v*';
```

Comment:
what's the reason of the change to `*v*` for the version mode? It was empty before and I would rather not change the default behavior if possible, did you see if `*v*` was used anywhere else and that's why you added it?

Reply:
Because  want to support two modes, if it empty, then can only make it effective in independent mode, otherwise there will be bugs in non-independent mode. 

the default setting is `*v*`, because in non-independent mode, the default tag is `v *.*.*`

Reply:
but where did you find that it`s the default for version mode? Even if `v` is the default tag, the user can change it via [`--tag-version-prefix`](https://github.com/lerna-lite/lerna-lite/tree/main/packages/version#--tag-version-prefix) and I don't want to break that, I really don't think `*v*` is really the default unless proven

Reply:
Sorry, I'm wrong.  if it empty, there won't be any bugs, but there won't be any left.

Reply:
Sorry, I'm wrong. There will be no bugs, just the same effect as the original code

Reply:
Sorry I'm not sure to follow what you mean, I still think it should be empty string for version mode (when not using independent mode). Could you add the missing yargs devDeps so that we could at least see the unit tests passing, I wrote a comment earlier

Reply:
yargs devDeps  is added, and  empty string for version mode is ok now